server/controllers/collection.controller/updateCollection.js

```diff
@@ -43,7 +43,7 @@ export default function createCollection(req, res) {
         { path: 'owner', select: ['id', 'username'] },
         {
           path: 'items.project',
-          select: ['id', 'name', 'slug'],
+          select: ['id', 'name', 'slug', 'visibility'],
```

Comment:
Annoying that you had to update this same block of code in 5 places.  That's definitely a sign of something that should be refactored.